src/LinePlot/LinePlot.tsx

```diff
@@ -52,7 +52,7 @@ const LinePlot = ({
         height="100%"
         width="100%"
       >
-        {showTitle ? (
+        {!isFullscreen && showTitle ? (
```

Comment:
What's this change here all about? I don't see any mention in the PR description of changes to LinePlot. I assume you have fixed a bug or something but you should mention it in the PR if you touch other components.

Reply:
Yes it was a bug fix, I have mentioned it in the PR now.